compiler+runtime/src/cpp/jank/jit/processor.cpp

```diff
@@ -199,4 +238,54 @@ namespace jank::jit
     auto err(interpreter->ParseAndExecute({ s.data(), s.size() }));
     llvm::logAllUnhandledErrors(std::move(err), llvm::errs(), "error: ");
   }
+
+  option<native_persistent_string>
+  processor::find_dynamic_lib(native_persistent_string const &lib) const
+  {
+    auto const &lib_name{ shared_lib_name(lib) };
+    for(auto const &lib_dir : library_dirs)
+    {
+      auto lib_abs_path{ fmt::format("{}/{}", lib_dir.string(), lib_name) };
+      if(boost::filesystem::exists(lib_abs_path.c_str()))
+      {
+        return lib_abs_path;
+      }
+    }
+
+    return none;
+  }
+
+  result<void, native_persistent_string>
+  processor::load_dynamic_libs(native_vector<native_persistent_string> const &libs) const
+  {
+    for(auto const &lib : libs)
+    {
+      auto const &lib_abs_path{ boost::filesystem::absolute(lib.c_str()) };
+      if(boost::filesystem::exists(lib_abs_path))
+      {
+        auto const &path_str{ lib_abs_path.string() };
+        load_object(path_str);
+      }
+      else
+      {
+        auto const result{ processor::find_dynamic_lib(lib) };
+        if(result.is_none())
+        {
+          return err(fmt::format("Failed to load dynamic library `{}`", lib));
+        }
+        else
+        {
+          load_object(result.unwrap());
+        }
+      }
+    }
+
+    return ok();
+  }
+
+  void processor::load_object(native_persistent_string const &path) const
+  {
+    std::cerr << "Loading dynamic library `" << path << "`\n";
```

Comment:
We don't want any logging by default. We may add a verbose mode to jank later, but for now we're silent.

Reply:
We do have some std:cerr's in this file. What's the purpose of them being there? Should I remove them?

Reply:
Ignore anything that was there. I'll be revamping all of the error reporting as my next task. But the happy path shouldn't be logging anything.